### PR TITLE
using hosted cluster name label filter the hosted works

### PR DESCRIPTION
**Pull request by skeeey**
Signed-off-by: Wei Liu <liuweixa@redhat.com>

### CODE REVIEW

File: pkg/controller/hosted/controller.go [hunk 1]
```diff
@@ -300,7 +300,8 @@ func createHostedManifestWork(managedClusterName string,
 			Name:      hostedKlusterletManifestWorkName(managedClusterName),
 			Namespace: manifestWorkNamespace,
 			Labels: map[string]string{
-				constants.HostedWorksLabel: "true",
+				constants.HostedWorksLabel:   "true",
```
Comment (zhujian7) [inline pkg/controller/hosted/controller.go:303]: do we still need this label?

Comment (skeeey) [reply pkg/controller/hosted/controller.go:303]: We need this, we use label to informer hosted works https://github.com/stolostron/managedcluster-import-controller/blob/main/cmd/manager/main.go#L171 to reduce the cache

Comment (skeeey) [reply pkg/controller/hosted/controller.go:303]: We can only check the hosted cluster label, will remove this

**Pull request merged**